### PR TITLE
Limits postgres privileges for application db users

### PR DESCRIPTION
**Pull request by acozine**
Related to #2735.

On the new postgresql infrastructure, users for specific databases should have limited privileges.

### CODE REVIEW

File: group_vars/figgy/production.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ application_dbuser_name: '{{figgy_db_user}}'
 application_dbuser_password: '{{figgy_db_password}}'
 application_dbuser_ro_name: '{{figgy_db_ro_user}}'
 application_dbuser_ro_password: '{{figgy_db_ro_password}}'
-application_dbuser_role_attr_flags: 'SUPERUSER'
+application_dbuser_role_attr_flags: 'CREATEDB'
```
Comment (kayiwa) [inline group_vars/figgy/production.yml:40]: Am I misremembering this being a problem for figgy?

File: group_vars/figgy/staging.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ application_dbuser_name: '{{figgy_db_user}}'
 application_dbuser_password: '{{figgy_db_password}}'
 application_dbuser_ro_name: '{{figgy_db_ro_user}}'
 application_dbuser_ro_password: '{{figgy_db_ro_password}}'
-application_dbuser_role_attr_flags: 'SUPERUSER'
+application_dbuser_role_attr_flags: 'CREATEDB'
```
Comment (kayiwa) [inline group_vars/figgy/staging.yml:34]: I think @hackartisan brought this up at an "office hours" meeting.

**Review by kayiwa — COMMENTED** [review]
We have an issue we need to verify with the figgy extensions https://github.com/pulibrary/princeton_ansible/issues/2780 Confirm that it won't break figgy before merging this.

**Comment by acozine** [comment]
Ah, right you are @kayiwa. This was an old branch I found on my local computer and I wondered why I still had it - but we were addressing these changes project by project.

I'll close the PR.

**Pull request closed without merging**